packages/labs/testing/src/lib/worker.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {parentPort, workerData} from 'worker_threads';
+import {render} from '@lit-labs/ssr/lib/render-with-global-dom-shim.js';
+
+import type {Payload} from './litSsrPlugin.js';
+
+if (parentPort === null) {
+  throw new Error('worker.js must only be run in a worker thread');
+}
+
+const {template, modules} = workerData as Payload;
+
+await Promise.all(modules.map((module) => import(module)));
+
+// Dangerously spoof TemplateStringsArray
+const strings = template.strings;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(strings as any).raw = strings;
```

Comment:
This won't work because `raw` property is readonly. I did try typing out a `SpoofableTemplateStringsArray` which removes the readonly.

Something like
```ts
type SpoofableTemplateStringsArray = {
  -readonly [P in keyof TemplateStringsArray]: TemplateStringsArray[P];
};

// or

interface SpoofableTemplateStringsArray extends TemplateStringsArray {
  raw: readonly string[];
}
```

Reply:
How about:

```ts
(strings as unknown as {raw: TemplateStringsArray['raw']})
```

Reply:
I like that. Typescript seems to also be okay without the `as unknown`.